OreNoTask/app/controllers/application_controller.rb

```diff
@@ -38,4 +38,9 @@ def render500(err = nil)
   def ensure_logged_in
     redirect_to login_url unless logged_in?
   end
+
+  def ensure_logged_in_admin
+    redirect_to login_url unless logged_in?
```

Comment:
38行目で定義した`ensure_logged_in`メソッドを使った方が良いと思いますー

```suggestion
    ensure_logged_in
```

Reply:
なるほど！！
修正しました。
https://github.com/Fablic/training/pull/1187/commits/cf30b15e46166521625c651b60c46e1c449ba274